crates/globset/src/serde_impl.rs

```diff
@@ -12,18 +13,62 @@ impl Serialize for Glob {
     }
 }
 
+struct GlobVisitor;
+
+impl<'a> Visitor<'a> for GlobVisitor {
+    type Value = Glob;
+
+    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        formatter.write_str("a string or a borrowed string")
```

Comment:
This should probably be "a glob pattern" or something more descriptive here.

Reply:
Thanks for noticing! There you go :)